File: PhysTasks/Task2.py
import math

g = 9.81

V = float(input("Введите начальную скорость (м/с): "))
alpha = float(input("Введите угол (в градусах): "))

alpha_rad = math.radians(alpha)

L = (V ** 2 * math.sin(2 * alpha_rad)) / g

print(f"Дальность полета L: {L:.2f} метров")

File: PhysTasks/Task2.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main() {
    const double g = 9.81; 
    
    double V, alpha;
    
    cout << "Введите начальную скорость (м/с): ";
    cin >> V;
    cout << "Введите угол (в градусах): ";
    cin >> alpha;

    double alpha_rad = alpha * M_PI / 180.0;

    double L = (V * V * sin(2 * alpha_rad)) / g;

    cout << "Дальность полета L: " << L << " метров" << endl;

    return 0;
}
